src/cpp/rtps/transport/ChainingTransport.cpp

```diff
@@ -0,0 +1,49 @@
+#include <fastdds/rtps/transport/ChainingTransport.h>
```

Comment:
I am missing the copyright and license notice

Reply:
Done